src/routers/chainbridge/Router.sol

```diff
@@ -0,0 +1,80 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.18;
+
+interface CentrifugeGatewayLike {
+    function handle(bytes memory message) external;
+}
+
+interface ChainBridgeLike {
+    function deposit(uint8 destinationChainID, bytes32 resourceID, bytes calldata data) external payable;
```

Comment:
Sorry if I am not seeing the obvious but what this name map to? I don't see a "deposit" function in the chainbridge pallet 🤔

Reply:
I just based it on the Bridge contract on Ethereum: https://etherscan.io/address/0xfe50ba7241b635eda23a32875c383a34e8a3596c#code (and Miguel gave a high level description of what would be required)

Reply:
Cool, thanks. I see the idea is that it deposits an Event to trigger the flow, that makes sense.